src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/newcontext--options--.md

```diff
@@ -0,0 +1,74 @@
+| options.screen.width                        | number  | `1280`                           | Page width in pixels.                                                                                                                                                                                                                             |
+| options.screen.height                       | number  | `720`                            | Page height in pixels.                                                                                                                                                                                                                            |
+| options.timezoneID                          | string  | system                           | Changes the context's timezone. See [ICU's metaZones.txt](https://cs.chromium.org/chromium/src/third_party/icu/source/data/misc/metaZones.txt?rcl=faee8bc70570192d82d2978a71e2a615788597d1) for a list of supported timezone IDs.                 |
+| options.userAgent                           | string  | browser                          | Specifies the user agent to use in the context.                                                                                                                                                                                                   |
+| options.viewport                            | object  | `{'width': 1280, 'height': 720}` | Sets a viewport size for all pages in the context. `null` disables the default viewport.                                                                                                                                                          |
+| options.viewport.width                      | number  | `1280`                           | Page width in pixels.                                                                                                                                                                                                                             |
+| options.viewport.height                     | number  | `720`                            | Page height in pixels.                                                                                                                                                                                                                            |
+
+</TableWithNestedRows>
+
+### Returns
+
+| Type   | Description                                                          |
+| ------ | -------------------------------------------------------------------- |
+| object | [BrowserContext](/javascript-api/xk6-browser/browsercontext/) object |
+
+
+### deviceScaleFactor example
```

Comment:
```suggestion
## deviceScaleFactor example
```

Reply:
Why are you suggesting this change? In other pages the example heading is `###`. It's actually confusing, is there a reason why it's generally `###` and not `##`?

Reply:
I noticed that in some places, it is `###`, while in others `##`.

For example, see [this](https://github.com/grafana/k6-docs/blob/b54ef018a2f875adbbac9b03c63ed835706b7279/src/data/markdown/docs/30%20xk6-browser/01%20xk6-browser/01%20Browser/close.md?plain=1#L11) and [this](https://github.com/grafana/k6-docs/blob/b54ef018a2f875adbbac9b03c63ed835706b7279/src/data/markdown/docs/30%20xk6-browser/01%20xk6-browser/01%20Browser/contexts.md?plain=1#L15).

Reply:
I'm going to make it the same everywhere, so i'll make it `##`.

Reply:
Having had a very quick look at the `k6` docs, they use `###`, so i'll make it the same as that to be consistent with the style.

Using `##` has a side affect which we may or may not want (see screenshot below with the red box to highlight the side affect). We should discuss this with the `k6` team as a whole since we should make it consistent across the whole of the documentation.

<img width="1084" alt="Screenshot 2022-09-08 at 12 00 59" src="https://user-images.githubusercontent.com/1112428/189106447-01062430-3654-4e01-95dd-93e50f676449.png">